### PR TITLE
Everest 240 list backups

### PR DESCRIPTION
**Pull request by fabio-silva**
https://jira.percona.com/browse/EVEREST-240

[negative request of storage locations.webm](https://github.com/percona/percona-everest-frontend/assets/90199600/f9d62228-ed18-468d-b25e-14072fece658)

[positive request of storage location.webm](https://github.com/percona/percona-everest-frontend/assets/90199600/fbfa3565-b1da-4113-bc45-81e26406c67d)

### CODE REVIEW

File: apps/everest/pages/backups/backups-list/backups-list.tsx [hunk 1]
```diff
@@ -1,5 +1,71 @@
...
+    []
+  );
+
+  const handleManualBackup = (handleClose: () => void) => {
+    handleClose();
+  }
+
+  return (
+    <Table
+      noDataMessage="You don't have any backups yet. Create one to get started"
```
Comment (filipmikes1) [inline apps/everest/pages/backups/backups-list/backups-list.tsx:56]: Can we move this to messages file?

File: apps/everest/pages/backups/backups-list/backups-list.tsx [hunk 2]
```diff
@@ -1,5 +1,71 @@
...
+      data={backups}
+      columns={columns}
+      renderTopToolbarCustomActions={() => (
+        <MenuButton buttonText='Create Backup'>
```
Comment (filipmikes1) [inline apps/everest/pages/backups/backups-list/backups-list.tsx:60]: Also messages file

File: apps/everest/pages/backups/backups-list/backups-list.tsx [hunk 3]
```diff
@@ -1,5 +1,71 @@
...
+          {/* MUI Menu does not like fragments and asks for arrays instead */}
+          {(handleClose) => [
+              <MenuItem key="now" onClick={() => handleManualBackup(handleClose)}>Now</MenuItem>,
+              <MenuItem key="schedule">Schedule</MenuItem>
```
Comment (filipmikes1) [inline apps/everest/pages/backups/backups-list/backups-list.tsx:64]: Messages

File: apps/everest/pages/backups/backups-list/backups-list.tsx [hunk 4]
```diff
@@ -1,5 +1,71 @@
...
+  return (
+    <Table
+      noDataMessage="You don't have any backups yet. Create one to get started"
+      data={backups}
+      columns={columns}
+      renderTopToolbarCustomActions={() => (
+        <MenuButton buttonText='Create Backup'>
+          {/* MUI Menu does not like fragments and asks for arrays instead */}
+          {(handleClose) => [
+              <MenuItem key="now" onClick={() => handleManualBackup(handleClose)}>Now</MenuItem>,
```
Comment (filipmikes1) [inline apps/everest/pages/backups/backups-list/backups-list.tsx:63]: Messages

File: apps/everest/pages/new-database/new-database.constants.ts [hunk 5]
```diff
@@ -16,19 +16,20 @@ import { DbType } from '@percona/ui-lib.db-toggle-card';
...
   // [DbWizardFormFields.pitrEnabled]: true,
   // [DbWizardFormFields.pitrTime]: '60',
-  // [DbWizardFormFields.storageLocation]: '',
-  // [DbWizardFormFields.selectedTime]: TimeValue.hours,
-  // [DbWizardFormFields.minute]: 0,
-  // [DbWizardFormFields.hour]: 12,
-  // [DbWizardFormFields.amPm]: AmPM.AM,
-  // [DbWizardFormFields.weekDay]: WeekDays.Mo,
-  // [DbWizardFormFields.onDay]: 1,
+  [DbWizardFormFields.storageLocation]: null,
```
Comment (filipmikes1) [inline apps/everest/pages/new-database/new-database.constants.ts:26]: Probably can also be empty string as in previous PR?

File: apps/everest/hooks/api/db-cluster/useDbCluster.ts [hunk 6]
```diff
@@ -35,19 +43,19 @@ const formValuesToPayloadMapping = (dbPayload: DbWizardType): DbCluster => {
...
-      //   }),
-      // },
+      backup: {
+        enabled: dbPayload.backupsEnabled,
+        ...(dbPayload.backupsEnabled && {
+          schedules: [
+            {
+              enabled: true,
+              name: '',
+              backupStorageName: typeof dbPayload.storageLocation === 'string' ? dbPayload.storageLocation : dbPayload.storageLocation!.name,
```
Comment (filipmikes1) [inline apps/everest/hooks/api/db-cluster/useDbCluster.ts:53]: What is the reason for this? When can dbPayload.storageLocation be string?

File: apps/everest/pages/new-database/new-database.constants.ts (hunk 5 above)
Comment (fabio-silva) [reply apps/everest/pages/new-database/new-database.constants.ts]: 👍

**Review by filipmikes1 — APPROVED** [review]

**Review by filipmikes1 — APPROVED** [review]

**Pull request merged**